packages/filesystem/src/browser/file-tree/file-tree-model.ts

```diff
@@ -61,6 +65,10 @@ export class FileTreeModel extends TreeModelImpl implements LocationService {
         }
     }
 
+    async drives(): Promise<URI[]> {
+        return this._drives.promise;
```

Comment:
`return (await this.fileSystem.getDrives()).map(uri =>  new URI(uri))` - not sure why it is necessary to reuse the same promise?

Reply:
It is not necessary, but during the lifecycle of the "File Dialog", I don't think we have to handle the "new drive was detected" events. Anyway, I do not mind always resolving it.